src/LiveComponent/assets/src/clone_html_element.ts

```diff
@@ -1,6 +1,6 @@
-export function cloneHTMLElement(element: HTMLElement): HTMLElement {
+export function cloneHTMLElement(element: HTMLElement|SVGElement): HTMLElement|SVGElement {
     const newElement = element.cloneNode(true);
-    if (!(newElement instanceof HTMLElement)) {
+    if (!(newElement instanceof HTMLElement) && !(newElement instanceof SVGElement)) {
         throw new Error('Could not clone element');
     }
```

Comment:
NOT cloning in the case of an SVG could cause unexpected problems (in some cases we clone because we then modify the cloned version, thinking that this is safe because it is not the REAL version on the page).

I think that we should use `element: HTMLElement` as the first argument and `HTMLElement` as the return type. Then, we'll rely on Typescript to tell us if we're being untruthful and are passing something potentially different into this method (though our Typescript still needs a lot of cleanup).

If we do this, there is only one place we will need to update to avoid Typescript being mad. It is in `live_controller` `_updateModelFromElement()`. This function should, I suppose, allow `element: Element` as the first arg. Then, on the first line, check if `!(element instanceof HTMLElement)`, then throw an exception.